rules/prefer-set-has.js

```diff
@@ -107,6 +107,35 @@ const isIncludesCall = node => {
 	);
 };
 
+const multipleCallNodeTypes = new Set([
+	'ForOfStatement',
+	'ForStatement',
+	'WhileStatement',
+	'DoWhileStatement',
+	'FunctionDeclaration',
+	'FunctionExpression',
+	'ArrowFunctionExpression',
+	'ObjectMethod',
+	'ClassMethod'
+]);
```

Comment:
These types are all I got in mind could call `.includes()` multiple times. Maybe more from other parser, need feedback.

Reply:
Forgot `ForInStatement`, added in https://github.com/sindresorhus/eslint-plugin-unicorn/pull/706/commits/aaa3638f7fdc48d4d283df6bc6595160032f45eb.